src/components/intel_gpu/README.md

```diff
@@ -1,48 +1,56 @@
-# Intel GPU Component: (intel_gpu)
+# Intel GPU Component (intel_gpu):
 
 PAPI "intel_gpu" component is for accessing Intel Graphics Processing Unit (GPU) hardware performance metrics through Intel oneAPI Level Zero Interface.
 
-## Enable intel_gpu component
+## Enable intel_gpu Component
 
 To enable intel_gpu component, the PAPI library is built with configure option as 
 
-	./configure --with-components="intel_gpu"
-
-It is requred to build with Intel oneAPI Level Zero header files. The directory of Level0 header files can be defined using INTEL_L0_HEADERS. Default installation location is /usr/include/level_zero/.
-
+    ./configure --with-components="intel_gpu"
 
 ## Prerequisites 
 
 * [oneAPI Level Zero loader (libze_loader.so)](https://github.com/oneapi-src/level-zero)
-* [Intel(R) Metrics Discovery Application Programming Interface](https://github.com/intel/metrics-discovery)
+* [Intel(R) Metrics Discovery Application Programming Interface (libigdmd.so)](https://github.com/intel/metrics-discovery)
+* [Intel(R) Metrics Library Application Programming Interface (libigdml.so)](https://github.com/intel/metrics-library)
 * /proc/sys/dev/i915/perf_stream_paranoid is set to "0"
 * User needs to be added into Linux render/video groups
 
-## Runtime environment:
+## Environment Variables
+
+PAPI requires the location of the Level Zero install directory. This can be
+specified by one environment variable: **PAPI\_INTEL\_GPU\_ROOT**.
+
+Default installation location is /usr.
+    
+Access to this directory is required at both compile (for include files) and
+at runtime (for libraries).
+
+## Runtime Environment:
 
-*  ```sh
-   export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<path-to-libze_loader>:<path-to-libmd>
-   ```
+*   ```sh
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<path-to-libze_loader>:<path-to-libigdmd>:<path-to-libigdml>
```

Comment:
Q: On Headroom at Oregon (Intel Data Center Max 1100), I did not set a path to `libigdmd.so` for the intel_gpu component to be active. What cases is this needed to be explicitly set?